### PR TITLE
CNDIT-1316: Adding helm charts for the investigation-reporting-service and post-processing-reporting-service

### PR DESCRIPTION
**Pull request by alog-enquizit**
CNDIT-1316: Adding helm charts for the investigation-reporting-service and post-processing-reporting-service

Wired the entrypoints from the modernization-api

### CODE REVIEW

File: charts/modernization-api/values-dts1.yaml
Comment (szamfir) [inline charts/modernization-api/values-dts1.yaml:37]: In order to prevent other environment from breaking you will also need to add lines 36-37 to all other values-*.yaml (to be safe).

**Review by szamfir — CHANGES_REQUESTED** [review]
See comment for change requested

File: charts/modernization-api/values-dts1.yaml
Comment (alog-enquizit) [reply charts/modernization-api/values-dts1.yaml:37]: Updated in all other places. Thanks for pointing out.

**Review by szamfir — APPROVED** [review]
LGTM, let me know once you merge so I can do a quick sanity check on environments

**Pull request merged**
